Git/gitcommit.sublime-syntax

```diff
@@ -7,66 +7,198 @@ file_extensions:
   - COMMIT_EDITMSG
   - MERGE_MSG
   - TAG_EDITMSG
-scope: text.gitcommit
+scope: text.git.commit
+
+variables:
+  hash: \b\h{7,}\b
+  # Rebase operations
+  shortcut: '[defprsx]'
+  operator: \b(?:drop|edit|exec|fixup|pick|reword|squash|{{shortcut}})\b
+  # The following variables are required to support highlighting of special phrases
+  # in the git commit messages for all supported languages.
+  # Source: https://github.com/git/git/tree/master/po
+  # Languages: bg,ca,de,fr,is,it,ko,pt_PT,ru,sv,vi,zh_CH
+  on_branch: On branch|На клон|En la branca|Auf Branch|Sur la branche|Sul branch|현재 브랜치|No ramo|На ветке|På grenen|Trên nhánh|位于分支
+  date: Date|Дата|Data|Datum|Date|@is.po|@it.po|시각|Data|Дата|Datum|Ngày tháng|日期
+  # file state variables
+  #  - @it.po , @is.po means no translation in the file found
```

Comment:
Shouldn't those be removed from the variables below? Since they are the same for all the "sections", you can't differentiate between them.

Otoh, they probably won't be used and instead the original english text is shown, so it's not like it will hurt.

Reply:
Those @it.po won't be added to a git generated comment and will therefore be ignored. We can use different placeholders if there are concerns about `@` being used as username reference, but I didn't see such in comments. too.

Each variable has the same amount of "columns". Each column references a language / po-file. By removing those values we will loose this reference. If a single translation changes, you'll have to go through all translations and built up the list again. At the moment you now, bg.po belongs to the second column and so on. There are even duplicates in the columns, which I did not eliminate in order to keep this reference.

We can't use `(?x)` to split up in multiple lines and add a comment to each translation without adding dozens of`[ ]` for whitespace and bloat the lists.